src/app/components/cron-gen/cron-gen.jsx

```diff
@@ -109,59 +128,66 @@ function ChplCronGen(props) {
   useEffect(() => updateCron(), [days, formik.values.runTime]);
 
   return (
-    <div className={classes.content}>
```

Comment:
If you remove the use of this class, that's fine. But you need to remove the class itself as well. Also, I noticed the class itself was updated. I know it's hard to find CSS that doesn't matter, but it's really important that we do, because if we don't it becomes even more difficult to modify things later, as we never know when style rule is important, and which isn't